### PR TITLE
Translate the 'parrot' adventure to German

### PR DESCRIPTION
**Pull request by DerSibi**
**Description**

I provide a German translation of the 'Parrot' adventure to German. Also, I take over the latest changes to the English language file to have them reflected in the German translations

**Fix for**

Describe what issue or discussion your pr concerns

**How to test**

Describe how to run your code to see it is action. See this https://github.com/Felienne/hedy/pull/880#issue-1016304308 for an example.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

### CODE REVIEW

**Comment by DerSibi** [comment]
@krulls: In case you woud like to have a look. Still cannot assign you as reviewer (when I enter your name in the search box, you won't show up there)

File: coursedata/adventures/de.yaml [hunk 1]
```diff
@@ -320,71 +320,71 @@ adventures:
...
+                start_code: "print Ich bin Hedy der Papagei!"
             2:
                 story_text: |
-                    ## Parrot
-                    Create your own online pet parrot that will copy you!
+                    ## Papagei
+                    Lass Deinen zahmen Papageien noch etwas echter klingen!
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
```
Comment (Felienne) [inline coursedata/adventures/de.yaml:346]: I know you are translating from the current English version, so you are not to blame for this, but we are in the middle of making a change to the yaml format, almost ready to be merged (see #1838), where example code will be explicitly labeled with a key in yaml, like this:

```suggestion
                example_code: |
                    ## Hedy Beispiel-Code
```

I can totally fix that for this PR of course, but if you go about translating new ones, maybe you can stick to this new format already just to save me a bit of work? Thanks!!

**Review by Felienne — APPROVED** [review]

File: coursedata/adventures/de.yaml (hunk 1 above)
Comment (DerSibi) [reply coursedata/adventures/de.yaml:346]: Hey,
alright, will take care of it for the next PR.

File: coursedata/adventures/de.yaml [hunk 2]
```diff
@@ -320,71 +320,71 @@ adventures:
...
-                    print Train your parrot!
-                    new_word is ask Which word do you want to teach them?
-                    add new_word to words
-                    print You: Say new_word , Hedy!
-                    print Hedy the Parrot: words at random
+                    Wörter is Aaaaaaak, Hedy
+                    print Bring Deinem Papageien etwas bei!
+                    Neues_Wort is ask Welches Wort willst Du ihm beibringen?
+                    add Neues_Wort to Wörter
+                    print 🧒 Sag Neues_Wort , Hedy!
```
Comment (krulls) [inline coursedata/adventures/de.yaml:368]: I like the usage of the emojis here but it is contradictory to the english version. fine for me, anyway.

**Review by krulls — COMMENTED** [review]
:+1:

File: coursedata/adventures/de.yaml (hunk 2 above)
Comment (Felienne) [reply coursedata/adventures/de.yaml:368]: yes we just recently started to allow emoji so I guess they will seep into content slowly

**Pull request merged**